addons/project_todo/static/src/scss/todo.scss

```diff
@@ -3,7 +3,6 @@
         display: flex;
         flex-direction: column;
         .o_form_statusbar {
-            margin: 0;
```

Comment:
Not sure why this change is needed, the visual result seems the same.

Reply:
This was not supposed to be here, it's related to the statusbar task